app/routes/news._index.mdx

```diff
@@ -26,6 +26,20 @@ import { Anchor } from '~/components/Anchor'
 
 ## 2024
 
+    <CollectionItem
+      className="maxw-none"
+      variantComponent={
+        <CollectionCalendarDate datetime={'June 14, 2024'} />
+      }
+    >
+      <CollectionHeading headingLevel="h3">
+        <Anchor>Circulars are Now Available via Kafka</Anchor>
+      </CollectionHeading>
+      <CollectionDescription>
+        Circulars can now be streamed over Kafka in JSON format on the topic `gcn.circulars`.  They can be added to your existing Kafka streaming codes or generated via the [QuickStart interface](/quickstart).
```

Comment:
- Eliminate passive voice.
- Don't refer to it as the "QuickStart interface"; we never use the term "QuickStart" except in the URL.